src/routes/boot.ts

```diff
@@ -361,11 +386,12 @@ const anonymousBoot = async (
   middleware?: BootMiddleware,
   shouldLogout = false,
 ): Promise<AnonymousBoot> => {
-  const [visit, flags, extra, firstVisit] = await Promise.all([
+  const [visit, flags, extra, firstVisit, exp] = await Promise.all([
     visitSection(req, res),
     getUserFeatureFlags(req, con),
     middleware ? middleware(con, req, res) : {},
     getAnonymousFirstVisit(req.trackingId),
+    getExperimentation(req.trackingId),
```

Comment:
Just wondering does growthbook allow the conversion from trackingId to userId eventually?

Reply:
Ideally they should be the same. We use `trackingId` as `userId` once the user registers